Meta/notify_irc.py

```diff
@@ -96,7 +96,8 @@ def run_on(json_string):
     wrapper = json.loads(json_string)
     line = compute_lines(wrapper)
     if line:
-        send_notification(line)
+        # send_notification(line)
+        print("About to send {}".format(line))
```

Comment:
Hmm?

Reply:
Ping. This reads like it would break the IRC notifications.

Reply:
reverted, sorry bout the delay!